local_planner/src/nodes/local_planner.cpp

```diff
@@ -16,11 +16,10 @@ LocalPlanner::~LocalPlanner() {}
 void LocalPlanner::setPose(const Eigen::Vector3f& pos,
                            const Eigen::Quaternionf& q) {
   position_ = pos;
-  curr_yaw_fcu_frame_deg_ = getYawFromQuaternion(q);
-  curr_yaw_histogram_frame_deg_ = -curr_yaw_fcu_frame_deg_ + 90.0f;
-
-  curr_pitch_fcu_frame_deg_ = getPitchFromQuaternion(q);
-  star_planner_->setPose(position_, curr_yaw_histogram_frame_deg_);
+  fov_.azimuth_deg = -getYawFromQuaternion(q) + 90.0f;
+  wrapAngleToPlusMinus180(fov_.azimuth_deg);
```

Comment:
could we do that in one line? e.g 
`fov_.azimuth_deg = wrapAngleToPlusMinus180(-getYawFromQuaternion(q) + 90.0f);`

and maybe put a comment above that those lines transform the yaw and pitch angles in fcu frame to the histogram convention

Reply:
nope, you can't modify an rvalue ;)

I can add the comments

Reply:
ok fine leave it the was it is, I did not look at function close enough. I just looked a bit weird to me :)

Reply:
I'd prefer if we could get these returning values, rather than doing it silently on references. It allows chaining functions for exactly things like his, making a lot of otherwise complex code easy to do, and it makes it easier to reformulate code into 'pipelines'.